packages/web/pages/learn/thegreathouses/[questchain].tsx

```diff
@@ -0,0 +1,164 @@
+import { Box, Spinner, Text, VStack } from '@metafam/ds';
+import { graphql } from '@quest-chains/sdk';
+import { PageContainer } from 'components/Container';
+import { MetaLink } from 'components/Link';
+import { Chain } from 'components/QuestChain/Chain';
+import Heading from 'components/QuestChain/QuestHeading';
+import { HeadComponent } from 'components/Seo';
+import { useWeb3 } from 'lib/hooks';
+import {
+  useLatestQuestChainData,
+  useLatestQuestStatusesForUserAndChainData,
+  useUserProgress,
+  useUserStatus,
+} from 'lib/hooks/questChains';
+import { GetStaticPaths, GetStaticPropsContext } from 'next';
+import React, { useCallback } from 'react';
+import { FaArrowLeft } from 'react-icons/fa';
+import { errorHandler } from 'utils/errorHandler';
+import {
+  QuestChainGreatHousesDetails,
+  QuestChainsGreatHouses,
+  QuestChainType,
+} from 'utils/questChains';
+
+const { getQuestChainInfo } = graphql;
+
+type Props = {
+  questChain: graphql.QuestChainInfoFragment;
+  name: QuestChainType;
+};
+
+const QuestChainPathPage: React.FC<Props> = ({
+  questChain: inputQuestChain,
+  name,
+}) => {
+  const { address } = useWeb3();
+
+  const {
+    questChain,
+    fetching: fetchingQuests,
+    refresh: refreshQuests,
+  } = useLatestQuestChainData(inputQuestChain);
+
+  const {
+    questStatuses,
+    fetching: fetchingStatus,
+    refresh: refreshStatus,
+  } = useLatestQuestStatusesForUserAndChainData(
+    inputQuestChain.chainId,
+    inputQuestChain.address,
+    address,
+  );
+
+  const fetching = fetchingStatus || fetchingQuests;
+
+  const refresh = useCallback(() => {
+    refreshStatus();
+    refreshQuests();
+  }, [refreshQuests, refreshStatus]);
+
+  const userStatus = useUserStatus(questStatuses);
+
+  const { progress, canMint } = useUserProgress(
+    address,
+    questChain,
+    userStatus,
+  );
+
+  if (!questChain) {
+    return (
+      <PageContainer>
+        <Text> Quest Chain not found! </Text>
+      </PageContainer>
+    );
+  }
+
+  return (
+    <PageContainer>
+      <HeadComponent
+        title={`MetaGame ${inputQuestChain.name}`}
+        description="MetaGame is a Massive Online Coordination Game! MetaGame has some epic quests going on!"
+        url="https://my.metagame.wtf/quests/name-of-the-engaged"
+      />
+      <VStack spacing={8} w="100%" maxW="96rem" align="stretch">
+        <Box w="100%">
+          <MetaLink href="/thegreathouses">
+            <FaArrowLeft
+              fontSize="0.875rem"
+              style={{ display: 'inline-block', marginRight: '0.5rem' }}
+            />
+            Back to the Great Houses
+          </MetaLink>
+        </Box>
+        <Heading
```

Comment:
In this component, when I click on the "created by" eth address, it doesn't go anywhere

Reply:
really weird, even the deployed version opens the profile in the new tab:
https://user-images.githubusercontent.com/35112939/208112258-eca4c78e-d8c3-4c44-b02d-beb2ddb25cfc.mov

Reply:
Huh, yeah, seems to work now. Strange